plugins/kubernetes/CHANGELOG.md

```diff
@@ -1,5 +1,15 @@
 # @backstage/plugin-kubernetes
 
+## 0.7.10-next.2
+
+### Patch Changes
+
+- Updated dependencies
+  - @backstage/core-plugin-api@1.5.1-next.1
+  - @backstage/plugin-catalog-react@1.4.1-next.2
+  - @backstage/core-components@0.12.6-next.2
+  - @backstage/config@1.0.7
```

Comment:
The main cause of all these bumps seems to be:
* the small docs update to `@backstage/config` : #16593. `@backstage/config` is depended on directly and indirectly by the other 3
* `@backstage/core-plugin-api` got updated for Typescript 5: #17094

Reply:
Not silly at all! It's definitely the case that there are version bumps that we could skip, it's just that it's some pretty tricky manual work to figure out which bumps would be safe to skip, as opposed to ones that must happen for the prerelease of the plugin to function as desired. It's ofc also due to the prereleases in general as we're bumping packages to make sure that they can be tested together with prerelease dependencies, which wouldn't be necessary if we only did mainline releases.